src/igvfd/loadxl.py

```diff
@@ -442,6 +442,7 @@ def attachment(path):
     from PIL import Image
     from base64 import b64encode
 
+    mimetypes.add_type('text/plain', '.yaml')
```

Comment:
@keenangraham We don't need this line any more right? We are anyways leaning towards just having .yaml submitted as .txt.